docs/src/developing/programming-model/runtime.md

```diff
@@ -115,16 +113,14 @@ Budget](#compute-budget).
 
 With a transaction-wide compute budget the `max_units` cap is applied to the
 entire transaction rather than to each instruction within the transaction. The
-default number of maximum units remains at 200k which means the sum of the
+default number of maximum units will be 1.4M which means the sum of the
 compute units used by each instruction in the transaction must not exceed that
-value. The number of maximum units allows is intentionally kept small to
-facilitate optimized programs and form the bases for a minimum fee level.
-
-There are a lot of uses cases that require more than 200k units
-transaction-wide. To enable these uses cases transactions can include a
-[``ComputeBudgetInstruction`](https://github.com/solana-labs/solana/blob/0224a8b127ace4c6453dd6492a38c66cb999abd2/sdk/src/compute_budget.rs#L44)
-requesting a higher compute unit cap. Higher compute caps will be charged
-higher fees.
+value. The default number of maximum units allows is intentionally kept large to
+avoid breaking existing client behavior.
+
+Most transactions won't require the maximum units so they can include a
+[``ComputeBudgetInstruction`](https://github.com/solana-labs/solana/blob/db32549c00a1b5370fcaf128981ad3323bbd9570/sdk/src/compute_budget.rs#L39)
+to lower the compute unit cap. Lower compute caps will be charged lower fees.
```

Comment:
Should highlight this more... that the cap default to large and thus the tx will be charged a relatively large fee.  Developers should request a fee level that corresponds to what they intend to require and thus only play that minimum required for their use case